editions/tw5.com/tiddlers/messages/WidgetMessage_ tm-modal.tid

```diff
@@ -14,6 +14,15 @@ The "currentTiddler" variable is set to the title of the modal tiddler, but can
 
 The modal message is usually generated with the ButtonWidget. The modal message is handled by the TiddlyWiki core.
 
+! The Modal Tiddler
+Within the Modal tiddler itself;
+
+*The field named "subtitle" will be used as the title of the Modal Window
```

Comment:
Modal Window should not be capitalised

Reply:
There should also be a space after the bullet point asterisk, and before the text